.gitignore

```
@@ -9,6 +9,9 @@ venv
 /pytest/**/__pycache__/
 /pytest/nearcore_pytest/nearcore_pytest.egg-info/
 /pytest/tests/test_contracts/parallel/target/
+tee_launcher/__pycache__/
+tee_launcher/__pycache__/
```

```suggestion
**/__pycache__/
/pytest/nearcore_pytest/nearcore_pytest.egg-info/
/pytest/tests/test_contracts/parallel/target/
```

Also, why not add a [proper Python-specific `.gitignore`](https://raw.githubusercontent.com/github/gitignore/master/Python.gitignore) in the directory where the Python code lives?